CHANGELOG.md

```diff
@@ -9,627 +9,627 @@
 
 ## Unreleased
 
-* Remove the inverse flag for contents list component (PR #1090)
-* Set all branded links to correct focus colour (PR #1088)
-* Fix components focus state spacing (PR #1054)
+* Remove the inverse flag for contents list component ([PR #1090](https://github.com/alphagov/govuk_publishing_components/pull/1090))
+* Set all branded links to correct focus colour ([PR #1088](https://github.com/alphagov/govuk_publishing_components/pull/1088))
+* Fix components focus state spacing ([PR #1054](https://github.com/alphagov/govuk_publishing_components/pull/1054))
 
 ## 19.0.0
 
-* **BREAKING:** Remove govuk_frontend_toolkit sass dependencies (PR #1069)
+* **BREAKING:** Remove govuk_frontend_toolkit sass dependencies ([PR #1069](https://github.com/alphagov/govuk_publishing_components/pull/1069))
     * This will break apps that rely on mixins and variables from govuk_frontend_toolkit - you must replace these with mixins and variables from GOV.UK Frontend
-* Restore margin bottom to the image card (PR #1079)
-* Allow summary_list to render without borders (PR #1073)
-* Explicitly set focus states (PR #1071)
-* Override edit link text on summary-link component (#1076)
-* Allow inset-text to take a block (PR #1078)
+* Restore margin bottom to the image card ([PR #1079](https://github.com/alphagov/govuk_publishing_components/pull/1079))
+* Allow summary_list to render without borders ([PR #1073](https://github.com/alphagov/govuk_publishing_components/pull/1073))
+* Explicitly set focus states ([PR #1071](https://github.com/alphagov/govuk_publishing_components/pull/1071))
+* Override edit link text on summary-link component ([PR #1076](https://github.com/alphagov/govuk_publishing_components/pull/1076))
+* Allow inset-text to take a block ([PR #1078](https://github.com/alphagov/govuk_publishing_components/pull/1078))
```

Comment:
This last one slipped in, but should actually be in Unreleased. I think it was because I may not have rebased against master before it was merged. Apologies!

Reply:
Good spot. Updated.